Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -313,6 +313,25 @@ theorem EuclideanSpace.piLpCongrLeft_single
 
 end DecEq
 
+section finAddEquivProd
+
+/-- The canonical linear homeomorphism between `EuclideanSpace 𝕜 (ι ⊕ κ)` and
+`EuclideanSpace 𝕜 ι × EuclideanSpace 𝕜 κ`. Note that this is not an isometry because
+product spaces are equipped with the supremum norm. -/
```

Comment:
```suggestion
`EuclideanSpace 𝕜 ι × EuclideanSpace 𝕜 κ`.

See `PiLp.sumPiLpEquivProdLpPiLp` for the isometry version,
where the RHS is equipped with the euclidean norm rather than the supremum norm. -/
```